tests/core/stdlib/stdlib_files.bzl

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@io_bazel_rules_go//go/private:providers.bzl", "GoSource")
+
+def _stdlib_files_impl(ctx):
+  files = ctx.attr._stdlib[GoSource].stdlib.files
+  runfiles = ctx.runfiles(files = files)
```

Comment:
ah, that's what I missed.
wouldn't it be simpler to add it to the `//:stdlib` rule ?

Reply:
I'd rather keep this in the `GoStdLib` provider for now, but I don't have a strong opinion about it, since this isn't a public interface. Fewer things to keep in sync though.